js_modules/dagit/packages/ui/src/components/configeditor/codemirror-yaml/mode.tsx

```diff
@@ -609,7 +619,14 @@ function findAutocompletionContext(
       const typeKey = nextTypeKey ? nextTypeKey : parentTypeDef?.configTypeKey;
       nextTypeKey = null;
 
-      const parentConfigType = schema.allConfigTypes.find((t) => t.key === typeKey)!;
+      let parentConfigType = schema.allConfigTypes.find((t) => t.key === typeKey)!;
+
+      // If nullable, extract the inner type.
+      if (parentConfigType.__typename === 'NullableConfigType') {
+        const innerType = parentConfigType.typeParamKeys[0];
+        parentConfigType = schema.allConfigTypes.find((t) => t.key === innerType)!;
+      }
+
```

Comment:
This seems like a safe change to me! Before we merge though, I'm curious if this blows up downstream if `parentConfigType!` is actually null. It seems like it was unsafe before and it's unsafe now, so probably fine 😅

Reply:
It seems possible, but yeah, it's already unsafe so I think this shouldn't cause new problems. I think if it does we'll spot it pretty quickly.